cmd/fyne/internal/mobile/env.go

```diff
@@ -326,6 +326,12 @@ func archNDK() string {
 		arch = "x86"
 	case "amd64":
 		arch = "x86_64"
+	case "arm64":
```

Comment:
What is this change for? it looks like it could break iOS build prior to Go 1.16

Reply:
This is for M1 macs, see https://golang.org/cl/346153.